Kickstarter-iOS/Views/SettingsTableViewHeader.swift

```diff
@@ -12,10 +12,11 @@ final class SettingsTableViewHeader: UITableViewHeaderFooterView {
   override func bindStyles() {
     super.bindStyles()
 
-    _ = self |> \.layoutMargins .~ .init(topBottom: Styles.grid(3), leftRight: Styles.grid(2))
+    _ = self
+      |> \.backgroundColor .~ .ksr_grey_200
```

Comment:
Since we've decided to use `UIView` (instead of `UITableViewHeaderFooterView` it won't inherit the table view's background and so we need to set it explicitly here).